src/styled_ppx.re

```diff
@@ -137,12 +259,12 @@ let moduleMapper = (_, _) => {
     switch (expr) {
     | {
         pmod_desc:
-          /* that are defined with the ppx [%styled] */
+          /* that are defined with a ppx like [%txt] */
           Pmod_extension((
-            {txt: "styled", _},
+            {txt, _},
```

Comment:
Hey @jchavarri, do you know if this is performant? I'm mapping thought all the ppx that have a text and later on, doing the split.

So I'm not sure if it's even a good idea. WDYT?

Reply:
I think it blocks other ppx to run actually 😢

Reply:
It shouldn't block other ppxs. I think the issue might be that the current code is "swallowing" the attributes of other ppxs.

If the ast returned respects other existing attributes and returns them, other ppxs should work ok 🤔